src/components/ToppingsSelect/index.jsx

```diff
@@ -1,19 +1,51 @@
 import React, { useState } from "react";
 import Topping from "../Topping";
-import './style.css';
+import './style.css'; 
 
 const ToppingsSelect = ({ toppings }) => {
+  const [cardToppings, setCardToppings] = useState(toppings);
+ 
+
+  const handleToppingChange = (index, newSelected) => {
+    const newToppings = [...cardToppings];
+    newToppings[index].selected = newSelected;
+    setCardToppings(newToppings);
+  } 
+  
+  const handlePriceChange = (index, newPrice) => {
+    const newPriceToppings = [...cardToppings];
+    newPriceToppings[index].price = newPrice;
+    console.log(newPrice);
+    setCardToppings(newPriceToppings);
+  }
+
+  let toppingSelected = 0;
+  cardToppings.forEach((topping) => toppingSelected += topping.selected);
+  
+
+  let toppingPrice = 0;
+  cardToppings.forEach((topping) => { 
```

Comment:
na obojí by se dalo použít filter a reduce ;) první
`const toppingSelected = cardToppings.filter((topping) => topping.selected).length;`
troufneš si na toppingPrice  pomocí reduce? Něco na způsob https://stackoverflow.com/questions/46680151/using-reduce-to-get-sum-of-all-numbers-in-an-array-in-javascript